src/wasm-lib/grackle/src/binding_scope.rs

```diff
@@ -21,6 +20,14 @@ pub enum EpBinding {
     Sequence(Vec<EpBinding>),
     /// A sequence of KCL values, indexed by their identifier.
     Map(HashMap<String, EpBinding>),
+    /// Not associated with a KCEP address.
+    Function(KclFunction),
```

Comment:
Interesting, I would've thought functions are assigned addresses so they can be passed around to other functions eventually!

Reply:
In the future we probably will need to do that. But right now, KCEP has no concept of a function, so it would never need to store a function in memory.